Xamarin.Forms.Core/RefreshView.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Windows.Input;
+using Xamarin.Forms.Platform;
+
+namespace Xamarin.Forms
+{
+	[ContentProperty("Content")]
+	[RenderWith(typeof(_RefreshViewRenderer))]
+	public class RefreshView : ContentView, IElementConfiguration<RefreshView>
+	{
+		readonly Lazy<PlatformConfigurationRegistry<RefreshView>> _platformConfigurationRegistry;
+
+		public RefreshView()
+        {
+            IsClippedToBounds = true;
+            VerticalOptions = LayoutOptions.FillAndExpand;
+            HorizontalOptions = LayoutOptions.FillAndExpand;
+			
+			_platformConfigurationRegistry = new Lazy<PlatformConfigurationRegistry<RefreshView>>(() => new PlatformConfigurationRegistry<RefreshView>(this));
+		}
+        
+        public static readonly BindableProperty IsRefreshingProperty =
+            BindableProperty.Create(nameof(IsRefreshing), typeof(bool), typeof(RefreshView), false);
+			
+        public bool IsRefreshing
+        {
+            get { return (bool)GetValue(IsRefreshingProperty); }
+            set
+            {
+                if ((bool)GetValue(IsRefreshingProperty) == value)
+                    OnPropertyChanged(nameof(IsRefreshing));
+
+                SetValue(IsRefreshingProperty, value);
+            }
+        }
+        
+        public static readonly BindableProperty CommandProperty =
+            BindableProperty.Create(nameof(Command), typeof(ICommand), typeof(RefreshView));
+			
+        public ICommand Command
+        {
+            get { return (ICommand)GetValue(CommandProperty); }
+            set { SetValue(CommandProperty, value); }
+        }
```

Comment:
You could use the lambda operator for trival get / set calls.

```
get => (ICommand)GetValue(CommandProperty);
set => SetValue(CommandProperty, value);
```

Other properties can use the same as well.